test/functional/dd-school-data-spec.js

```diff
@@ -18,7 +18,8 @@ fdescribe( 'The dynamic financial aid disclosure', function() {
     expect( page.totalProgramDebt.getText() ).toEqual( '23,000' );
   } );
 
-  it( 'should dynamically display the completion rate if it\'s available', function() {
+  /* Note: this item was removed from the settlement version of the code */
+  /* it( 'should dynamically display the completion rate if it\'s available', function() {
```

Comment:
Instead of commenting these out, change `it` to `xit` and leave uncommented (though if the implementation is no longer applicable, that could be deleted).